src/firefly/js/ui/tap/ObsCore.jsx

```diff
@@ -0,0 +1,752 @@
+import React,  {useState, useEffect, useReducer} from 'react';
+import PropTypes from 'prop-types';
+import Enum from 'enum';
+import {get, has, isUndefined, set} from 'lodash';
+import FieldGroupUtils, {getField, getFieldVal} from '../../fieldGroup/FieldGroupUtils.js';
```

Comment:
you can clean up some unused imports.

Reply:
Did a bit of cleanup on the code, I think this is resolved now.